sql-plugin/pom.xml

```diff
@@ -422,6 +422,24 @@
                 </dependency>
             </dependencies>
         </profile>
+        <profile>
+            <id>release340</id>
+            <activation>
+                <property>
+                    <name>buildver</name>
+                    <value>340</value>
+                </property>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>org.apache.orc</groupId>
+                    <artifactId>orc-core</artifactId>
+                    <version>1.8.0</version>
+                    <classifier>shaded-protobuf</classifier>
+                    <scope>provided</scope>
```

Comment:
Typically we're picking up ORC during the build as a transitive dependency of our Spark dependency.  We don't want to manually maintain this, otherwise when Spark moves its dependency we may not notice at compile time. 

I'm wondering why we need to be explicit about this dependency when we're not in any other case.  I would expect the Spark 3.4.0-SNAPSHOT artifacts to depend on the shaded-protobuf classifier versions of the ORC jars, and therefore those would be added to the classpath automatically as in other builds against Spark.  AFAICT we're not explicitly depending on ORC anywhere except the Databricks special case needed for that build environment.

If this is indeed necessary, this makes me wonder how this won't potentially conflict with whatever else was providing ORC in the classpath before this change.  If it wasn't through Spark, maybe we should be excluding ORC from one of our other dependencies rather than hoping this gets in front of the undesired ORC version in the classpath?

Reply:
Thanks for catching this, the main issue was actually a typo in the package name. The change to pom introduced another issue that masked but did not fix the original issue. It's hard to see real issues until the compile runs through successfully.